clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntity.java

```diff
@@ -202,6 +203,8 @@ public void releaseCache(String name) throws ClusteredTierReleaseException, Time
   public void destroyCache(String name) throws ClusteredTierDestructionException, TimeoutException {
     try {
       invokeInternal(timeouts.getLifecycleOperationTimeout(), messageFactory.destroyServerStore(name), true);
+    } catch (ResourceBusyException e) {
```

Comment:
So you want to differentiate between a busy and a fault simply through the message in the `ClusteredTierDestructionException` being thrown?  For a busy, using a different exception would permit a program to delay for a bit and retry the destroy; re-using the `ClusteredTierDestructionException` makes that more of a challenge.